tests/cross_functional/ui/fusion_access/test_create_filesystem.py

```
@@ -38,17 +40,70 @@ class TestFDFSANConnection(ManageTest):
+                "auths": {
+                    "cp.icr.io": {
+                        "username": "cp",
+                        "password": ent_key,  # pragma: allowlist secret
+                        "auth": auth_b64,
+                    }
+                }
+            }
+        )
+
+        # Create / update the entitlement-key secret
+        secret_name = "ibm-entitlement-key"
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to: "secret_name"

(S105)